plugin/table/table.go

```diff
@@ -71,6 +71,39 @@ func WithNewlineBehavior(behavior NewlineBehavior) option {
 	}
 }
 
+type CellPaddingBehavior string
+
+const (
+	// CellPaddingAligned adds visual padding to cells to make each column equal width (default).
+	CellPaddingAligned CellPaddingBehavior = "aligned"
+	// CellPaddingMinimal keeps a very small amount of padding to balance table readability while also reducing character count.
+	CellPaddingMinimal CellPaddingBehavior = "minimal"
+	// CellPaddingNone refrains from adding the padding to the cells.
+	CellPaddingNone CellPaddingBehavior = "none"
+)
+
+// WithPadColumns configures how to handle padding in table cells.
+// When set to "aligned" (default), every cell's text is padded to the width of the largest cell in its column.
+// When set to "minimal", every cell gets a space at the beginning and end of the cell for some minimal padding.
+// When set to "none", no extra padding is applied to cells.
+func WithCellPadding(behavior CellPaddingBehavior) option {
+	return func(p *tablePlugin) error {
+		switch behavior {
+		case "":
+			// Allow empty string to default to "aligned"
+			p.cellPaddingBehavior = CellPaddingBehavior(CellPaddingAligned)
```

Comment:
The wrapping with `CellPaddingBehavior()` is not needed here since it has already the correct type.